libft/sources/strings/strings_internal.h

```diff
@@ -6,45 +6,22 @@
 /*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/10/11 16:40:11 by taretiuk          #+#    #+#             */
-/*   Updated: 2024/10/11 16:40:15 by taretiuk         ###   ########.fr       */
+/*   Updated: 2024/11/04 05:42:37 by taretiuk         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #ifndef STRINGS_INTERNAL_H
 # define STRINGS_INTERNAL_H
 
-# include <stddef.h>
-# include <stdlib.h>
+# include "../libft/libft.h"
 # include "strings.h"
```

Comment:
Can you please remove these two includes?

Reply:
Not libft'h! Strings.h and memory.h, they're in libft already

Reply:
but for what to include all headers from libft in internal header if i can include just strings, memory and bool and it will work?

also, you commented 18th line and said these two, one above is libft.

Reply:
you included libft.h already anyway, so no need to duplicate these incliudes

in a really big project with hundreds of dependencies - yes, it makes sense to list all of them, without transitive dependencies. in a medium-sized like this - we might save energy and concentration by grouping includes into a meta-include (libft.h, tests_internal.h, minishell.h etc.)